gnucash/gnome/assistant-stock-transaction.cpp

```diff
@@ -262,7 +262,7 @@ static const TxnTypeVec long_types
         // Translators: this is a stock transaction describing a stock
         // split
         N_("Stock split"),
-        N_("Stock price is fractionally reduced")
+        N_("Company issues additional units, thereby reducing the stock price by a divisor, while keeping the $ value of shares constant.")
```

Comment:
Can we replace `$` here and later?
Reason: Languages are often shared by several countries with different currency symbols like en in GB (Lira Symbol), de in several EU  countries (€) and CH (CHF), …

Reply:
Good catch. Let's use `currency` instead.

Reply:
"while keeping the currency value of shares constant"? As a layperson I'd understand "while keeping the stock valuation constant"

Reply:
Well, if we're going to word-smith it:
Valuation can be construed as price, so you don't want that, and the distinction between `of shares` and `of a share` is maybe a bit fine. Plus you started with `units` so you should use that instead of `share` or `shares` throughout the sentence. `Issues units` isn't always right either, sometimes they take units away.

How about `Issuer increases or decreases the units held, proportionally adjusting the price so that the total value doesn't change.`?

Reply:
> Company issues additional units, thereby reducing the stock price by a divisor, while keeping the $ value of shares constant.

Instead, we could use the word `monetary` in place of the $ sign.

---
> Plus you started with units so you should use that instead of share or shares throughout the sentence.

In terms of units vs shares, the different text is purposeful.  The number of units values, but the value of the overall total shares does not change.  They should not be the same word.  Instead, we could refer to it as the `investment`.

---
> Issues units isn't always right either, sometimes they take units away.

Reverse split is covered in the next transaction, to this is exclusively a forward split and therefore exclusively issuing units.

---
New version:
> Company issues additional units, thereby reducing the stock price by a divisor, while keeping the **total monetary** value the **overall investment** constant.